analysis.py

```diff
@@ -143,34 +256,44 @@ def analyze(filename, initial_clusters, clusters_to_try, bins_to_try, max_L_to_t
     """
     if not os.path.isdir(save_dir):
         os.mkdir(save_dir)
-
+    
+    #read and preprocess data
     data = parsing.read_momentum(filename)
-    data = preprocess.molecular_frame(data[:10000, :])
+    data = preprocess.molecular_frame(data[:, :])
```

Comment:
We should add a parameter in .yml and argparse to allow for analysis of subsets of the data for testing purposes.